static/js/test-data.js

```diff
@@ -32,11 +32,11 @@ $(document).ready(function () {
         credentials: 'include'
     })
         .then((_res) => {
-            let token;
+            let iToken="";
```

Comment:
Remove the redundant `iToken` declaration form here since it is already declared globally.